robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -155,15 +184,23 @@ def validate_upcoming_schedules(self, num_payments, startdate, dayofmonth):
         installments = self.selenium.get_webelements(installmentrow)
         count = len(installments)
         print(f"Number of installments created is {count}")
-        assert count == int(num_payments), "Expected installments to be {} but found {}".format(num_payments, count)
+        assert count == int(
+            num_payments
+        ), "Expected installments to be {} but found {}".format(num_payments, count)
```

Comment:
suggestion, looks like u still use the linting tool can u adjust the formatting of this to make it readable